modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -20,39 +20,36 @@ However, to do so, you have to set `'sharing.federation.allowHttpFallback' => tr
 
 Follow these steps to establish a trusted connection between two servers.
 
-. Verify that both servers have SSL certificates.
-  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid.
+. Verify that both servers have SSL certificates
+  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid
 +
 .Lock icon in the address bars in Firefox, Google Chrome, and Safari.
 image:configuration/files/browser-address-bars.png[Lock icon in Firefox, Google Chrome, and Safari]
-. Verify that the `'overwrite.cli.url' => 'https://<SERVER_URL>' setting is configured to the correct URL, instead of `localhost`, in **config.php**.
-. Reset the federation job in your `oc_jobs` table. 
-  This job is required to get the verification token from the other server to establish a federation connection between two servers. 
-  The resetting ensures that it will be executed when we run xref:configuration/server/background_jobs_configuration.adoc#cron[system:cron] later.
+. Verify that the `'overwrite.cli.url' => 'https://<SERVER_URL>'` setting is configured to the correct URL instead of `localhost` in **config.php**
+. Navigate to **admin settings -> Sharing -> Federation**
+. Add **server 1** to the trusted servers on **server 2**
+. Add **server 2** to the trusted servers on **server 1**
+. Run `system:cron` from the ownCloud directory of both servers (for example `/var/www/owncloud/`)
```

Comment:
```suggestion
. Run `{occ-command-example-prefix} system:cron` in both servers
```

Wrong place to discuss if it is needed to really chdir somwhere, when running occ commands. (And it should not be needed to chdir in anycase, imo, I never do)

But occ should be mentioned, when running occ commands.

Reply:
I have further improved the text based on your changes, see commit, pls approve